migrations/versions/20b836df74cc_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 20b836df74cc
+Revises: 4cac94c86047
+Create Date: 2018-05-25 01:46:53.292127
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '20b836df74cc'
+down_revision = '4cac94c86047'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('microlocation')
```

Comment:
@bhaveshAn why are we dropping microlocation here ?

Please check your migrations properly before pushing.

Reply:
@niranjan94 Please review now.